contracts/tg4-group/src/contract.rs

```diff
@@ -100,62 +93,11 @@ pub fn execute_update_members(
     ];
 
     // make the local update
-    let diff = update_members(deps.branch(), env.block.height, info.sender, add, remove)?;
-    // call all registered hooks
-    let messages = HOOKS.prepare_hooks(deps.storage, |h| {
-        diff.clone().into_cosmos_msg(h).map(SubMsg::new)
-    })?;
-    Ok(Response::new()
-        .add_submessages(messages)
-        .add_attributes(attributes))
-}
-
-pub fn execute_add_hook(
-    deps: DepsMut,
-    info: MessageInfo,
-    hook: String,
-) -> Result<Response, ContractError> {
-    // Same as cw4-group guard: being admin
-    if !ADMIN.is_admin(deps.as_ref(), &info.sender)? {
-        return Err(Unauthorized {});
-    }
-
-    // add the hook
-    HOOKS.add_hook(deps.storage, deps.api.addr_validate(&hook)?)?;
-
-    // response
-    let res = Response::new()
-        .add_attribute("action", "add_hook")
-        .add_attribute("hook", hook)
-        .add_attribute("sender", info.sender);
+    update_members(deps.branch(), env.block.height, info.sender, add, remove)?;
```

Comment:
Let's put a FIXME here. Create an issue to re-introduce hooks, and link it here.